velox/dwio/parquet/tests/reader/ParquetReaderTest.cpp

```diff
@@ -1215,3 +1215,24 @@ TEST_F(ParquetReaderTest, testLzoDataPage) {
           .str(),
       "31232");
 }
+
+TEST_F(ParquetReaderTest, testEmptyDataPage) {
+  const std::string sample(getExampleFilePath("snappy.parquet"));
+
+  facebook::velox::dwio::common::ReaderOptions readerOptions{leafPool_.get()};
```

Comment:
nit: we should put the entire test suite under a facebook::velox::dwio::test` namespace so you don't need to specify the full path to everything.

Reply:
I fixed this to `dwio::common::ReaderOptions`. There are a couple of `using namespace ...` on top. This is now consistent with the `velox/dwio/dwrf/test/ReaderTest.cpp` test.  Both are not under any namespace.

Reply:
I cleaned up the remaining as well.